src/Compilers/Core/Portable/CodeGen/PermissionSetAttribute.cs

```diff
@@ -81,7 +81,7 @@ public ImmutableArray<Cci.IMetadataNamedArgument> GetNamedArguments(EmitContext
 
             // PermissionSetAttribute type must have a writable public string type property member 'Hex'
             Debug.Assert(((INamedTypeSymbol)_sourceAttribute.GetType(context)).GetMembers(HexPropertyName).Any(
-                member => member.Kind == SymbolKind.Property && ((IPropertySymbol)member).Type.SpecialType == SpecialType.System_String));
+                member => member is IPropertySymbol { Type: { SpecialType: SpecialType.System_String }, Kind: SymbolKind.Property }));
```

Comment:
>Kind: SymbolKind.Property [](start = 104, length = 25)

Semantically redundant.

Reply:
➡️ Treating this one as a duplicate of https://github.com/dotnet/roslyn/pull/39738#discussion_r344401827